src/main/java/ai/rev/speechtotext/clients/ApiClient.java

```diff
@@ -31,7 +31,7 @@
   private OkHttpClient client;
 
   /** Interface that ApiClient methods use to make requests */
-  public ApiInterface apiInterface;
+  public AsyncApiInterface asyncApiInterface;
```

Comment:
renaming any public member is a breaking change, right?

Reply:
Maybe these should be private?

Reply:
Unfortunately you cannot declare the class protected. But even if it were, that would make this class accessible only within the same package which would prevent the unit tests from using it :(